registrations/templates/signup_group_confirmation_to_waiting_list.html

```diff
@@ -0,0 +1,49 @@
+{% extends 'base_email.html' %}
+{% load i18n %}
+
+{% block content %}
+  <tr>
+    <td class="content-cell" style="padding:0;margin:0;text-align:left;">
+      <div class="content-container" style="padding:0 16px;">
+        <h1 style="color:#1a1a1a;font-size:36px;line-height:43px;font-weight:700;margin:0 0 12px;">
+          {% blocktranslate %}Thank you for signing up for the waiting list{% endblocktranslate %}
+        </h1>
+        <div class="divider" style="background:#0000bf;border-radius:0px;height:3px;width:100%;margin:24px 0;"></div>
+        <div style="margin: 56px 0px 40px">
+          <p style="color:#1a1a1a;font-size:16px;font-weight:normal;line-height:24px;margin:0px;">
+            <!-- Event -->
+            {% if event_type_id == "1" %}
+            {% blocktranslate %}The registration for the event <strong>{{event_name}}</strong> waiting list was successful.{% endblocktranslate %}
```

Comment:
SonarCloud is complaining about duplication due to these. It seems that the event display name, like "event", "course", etc, could be a variable that could be passed to a single `blocktranslate` string. Or the translated block could be passed completely as a variable to the template.

Reply:
I refactored confirmation email logic little bit and moved confirmation email template texts to a variable instead of having them in the templates. At the same time also removed unnecessary templates to make css and html consistent and easier to maintain.